tools/depends/target/gnutls/Makefile

```diff
@@ -3,14 +3,10 @@ DEPS= ../../Makefile.include Makefile size-max.patch add-dl-as-private-lib.patch
 
 # lib name, version
 LIBNAME=gnutls
-VERSION=3.6.11.1
+VERSION=3.6.13
```

Comment:
please bump to 3.6.14, since a serious security issue has been fixed: https://lists.gnupg.org/pipermail/gnutls-help/2020-June/004648.html